### ISSUE TITLE
Multi-tag support

### ISSUE THREAD

**Issue by ChaotikZ** [timusus/Shuttle#109]
Hey there, long time Shuttle user, this is something I've seen in other players but Shuttle seems to lack, so I'll get it out there.

If a music file has multiple tags, for example:
_Rock/Alternative Rock/Indie/Britpop_
or
_Jazz/Country/Indie_

Shuttle will create a "Rock/Alternative Rock/Indie/Britpop" and a "Jazz/Country/Indie" list in the genre section, instead of separating them like so:
_Rock
Alternative Rock
Indie
Britpop
Jazz
Country_

Can this be implemented somehow? It kind of defeats the purpose of using the genre section since most of the music I have has multiple tags.

Thanks a lot for all your hard work.

**Comment by thatguy7** [comment]
It seems like Android expects for the genres to be split by ";". This way the genres are recognized as separate entries, but each song is assigned just the first one. So if a mp3 is tagged as Rock;Blues, it will only show up in the Rock genre.

If Shuttle could do that properly, it would be very cool. Especially if one could do stuff like create playlists by specifying genres, eg. all that have genre Jazz, but none that also have genre Slow. I'd totally re-tag my library for this.

**Comment by sanyabeast** [comment]
Yes. Sadly, but it seems like this useful feature is missing in all music players in the world.
